public/index.html

```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link href='https://fonts.googleapis.com/css?family=Outfit' rel="stylesheet">
```

Comment:
these fonts are also available in _variables.scss

Reply:
We have copied the updates variable.scss and confirmed that it is okay to have the imports in index.html.